src/CodeGeneration/ApiGenerator/Overrides/Descriptors/RefreshDescriptorOverrides.cs

```diff
@@ -0,0 +1,23 @@
+using ApiGenerator.Domain;
+
+namespace ApiGenerator.Overrides.Descriptors
```

Comment:
Can we move the *.obsolete.json and *.patch.json files to a different directory to the downloaded specs? I think it'll make it easier to manage